build.gradle

```diff
@@ -50,6 +50,9 @@ subprojects {
       showStackTraces true
       showCauses true
     }
+
+    // Enabled dynamic agent loading (ByteBuddy) for JDK 21+ https://openjdk.org/jeps/451
+    jvmArgs '-XX:+EnableDynamicAgentLoading'
```

Comment:
Do consumers need to add this too?

(i.e. should this be part of the plugin, behind a version check: [`JavaVersion.current()`](https://github.com/TWiStErRob/net.twisterrob.gradle/blob/cff3808e335220f2ca9c3aa6803c8243b19a8123/docs/examples/local/settings.gradle.kts#L40-L42) or [`Test.javaVersion`](https://github.com/TWiStErRob/net.twisterrob.inventory/blob/d9c3f9dc2e80ae08fb83f606e436071fad8552e9/gradle/plugins-inventory/src/main/kotlin/net/twisterrob/inventory/build/allprojects.gradle.kts#L56-L70))

Reply:
good point let me update this 👍🏽

Reply:
What do you think about making it part of the Paparazzi Gradle Plugin? If I added Paparazzi to a project running CI on 22, would I need to add this line too?

Reply:
yep, I added this in my latest commit

Reply:
note: if you have it in plugin, then is it still necessary in root build.gradle?